textrepo-app/src/main/java/nl/knaw/huc/service/task/indexer/JdbiIndexFileTaskBuilder.java

```diff
@@ -189,11 +189,15 @@ public String run() {
                 () -> log.warn("No such mimetype: {}", m)
             );
           });
+      var msg = "";
```

Comment:
Voorstel: liever `final String msg;` ipv `var msg = "";`

Rationale: als je hier `final String msg;` zet ipv initialisatie op lege `""` om type `String` te krijgen, dan helpt de compiler met garanderen dat je in de volgende `if .. else ..` altijd iets in `msg` stopt. Mocht het nu in een refactoring per ongeluk verdwijnen, dan krijg je runtime een lege `""` ipv compile-time een warning.

Reply:
Opgelost in 4cf8cf7